BLSQ/iaso
# POLIO-1252 remove "reason" field from round history

deprecated, duplicate field for reason_for_delay

Related JIRA tickets : POLIO-1252

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes

- remove field from model + add migration
- remove from serializers
- remove from front-end

## How to test

Go to a campaign, edit a round date:
- Saving should be OK
- The reason inoput should be displayed correctly in round history

## Print screen / video

No changes

## Notes

The migration has already be done. It's the polio migration 0150.

## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
